### PR TITLE
Follow up to Update to gradio 5.5.0 #100

### PR DESCRIPTION
**Pull request by HeavyTony2**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #100 
Basically is the same than the former PR, but with correcting something for audioop-lts https://github.com/conda-forge/gradio-feedstock/pull/100#pullrequestreview-2459427770

### CODE REVIEW

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

**Comment by HeavyTony2** [comment]
@conda-forge-admin, please rerender

**Comment by Sclare87** [comment]
In my opinion the use of pydub in gradio is overengeneering. I dont think anybody need support for speech recogition in a "gui framework". Or it should be add as a addon.

Nevertheless thanks for the workaround.
